src/python_testing/TC_XX_1_1.py

```diff
@@ -0,0 +1,138 @@
+#
+#    Copyright (c) 2022 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import logging
+
+from matter_testing_support import MatterBaseTest, async_test_body, default_matter_test_main
+from mobly import asserts
+
+standard_range_prefix = 0x0000
+manufacturer_code_min_prefix = 0x0001
+manufacturer_code_max_prefix = 0xFFF0
+test_vendor_min_prefix = 0xFFF1
+test_vendor_max_prefix = 0xFFF4
+invalid_vendor_min_prefix = 0xFFF5
+invalid_vendor_max_prefix = 0xFFFF
+
+attribute_id_range_min = 0x0000
+attribute_id_range_max = 0x4FFF
+attribute_global_range_min = 0xF000
+attribute_global_range_max = 0xFFFE
+attribute_id_invalid_max = 0xFFFF
+
+# event_id_range_min = 0x00
+# event_id_range_max = 0xFF
+# event_id_invalid_max = 0xFFFF
+
+command_id_range_min = 0x00
+command_id_range_max = 0xFF
+command_id_invalid_max = 0xFFFF
+
+ATTRIBUTE_LIST_ID = 0xFFFB
+# EVENT_LIST_ID = 0xFFFA
+ACCEPTED_COMMAND_LIST_ID = 0xFFF9
+GENERATED_COMMAND_LIST_ID = 0xFFF8
+
+
+def get_element_prefix_suffix(element):
+    prefix_mask = ~0xffff
+    suffix_mask = 0xffff
+
+    prefix = (element & prefix_mask) >> 16
```

Comment:
How is this different from just:
```suggestion
    prefix = (element >> 16)
```
?

Reply:
Since it is bitshifting anyways, you are right. The mask dones not really do anything here.